source/_integrations/oncue.markdown

```diff
@@ -30,3 +30,11 @@ The Oncue by Kohler integration will allow you to monitor the state of your [Onc
 - [KG80R](https://kohlerpower.com/en/residential/generators/product/kg80r)
 
 {% include integrations/config_flow.md %}
+
+## Non-functional and Incorrect Entities
+
+Some entities do not function with some generators.  For example the 20RESA does not report engine coolant temperature, and it will show as 0C (32F).  Also enot related are engine compartment temperature, current, and all related (power) entities.  On the 20RESA at least, the total generated power entry appears bogus.
+
+## Polling Considerations
+
+Currently the integration polls every 10 minutes.   Polling faster would have fresher data, and also use more resources.  Also there was at one point a report of someone being banned (unknown if permanent) for polling too often (unknown how fast).
```

Comment:
You could lift this example with a small tweak 
https://www.home-assistant.io/integrations/hunterdouglas_powerview/#force-update-shade-position

Reply:
Could you post yaml for that, for me to steal as an example?

Also, could you comment on whether there have been any issues from 1 minute polling, and why you don't just run at 1 min always?  I would think it takes up to 10 minutes to start the 1-min polling, but maybe you trigger on something else too.